src/main/java/me/axieum/mcmod/authme/mixin/RealmsMainScreenMixin.java

```diff
@@ -0,0 +1,32 @@
+package me.axieum.mcmod.authme.mixin;
+
+import com.mojang.realmsclient.RealmsMainScreen;
+import net.minecraft.client.gui.screen.Screen;
+import net.minecraft.client.util.Session;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Mutable;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.gen.Accessor;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(RealmsMainScreen.class)
+public interface RealmsMainScreenMixin
+{
+    @Accessor(value="checkedClientCompatability")
+    @Mutable
+    static void setCheckedClientCompatability(boolean checkedClientCompatability){
+        // Method body is ignored.
+        throw new UnsupportedOperationException();
```

Comment:
I think we can simplify both of these down to be like [`SetSessionMixin`](https://github.com/Axieum/AuthMe/blob/master/src/main/java/me/axieum/mcmod/authme/mixin/SetSessionMixin.java)

Reply:
Done! :)